src/Microsoft.ML.Data/MLContext.cs

```diff
@@ -79,6 +79,13 @@ public sealed class MLContext : ISeededEnvironment
         /// </summary>
         public ComponentCatalog ComponentCatalog => _env.ComponentCatalog;
 
+        private static string _tempFilePath = System.IO.Path.GetTempPath();
```

Comment:
>System.IO [](http://example.com/codeflow?start=46&length=9)

does it make sense to add "using System.IO" in the top of the file?

Reply:
I didn't just because I am only using it in that one place. But I'm happy to switch it.